### PR TITLE
Task 7

### PR DESCRIPTION
**Pull request by cuthullu**

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ var todoListPlaceholder = document.getElementById("todo-list-placeholder");
 var form = document.getElementById("todo-form");
 var todoTitle = document.getElementById("new-todo");
 var error = document.getElementById("error");
+var todoCount = 0;
```
Comment (seanworkcode) [inline public/main.js]: I think this needs a more descriptive name, is it counting the todos or completed todos?

File: test/e2e.js [hunk 2]
```diff
@@ -86,5 +86,20 @@ testing.describe("end to end", function() {
             });
         });
     });
+    testing.describe("counting completed iems", function() {
```
Comment (seanworkcode) [inline test/e2e.js]: Good test!

**Comment by seanworkcode** [comment]
Only one update needed, then it's ready.

File: public/main.js [hunk 3]
```diff
@@ -79,6 +81,8 @@ function reloadTodoList() {
             completeBox.checked = todo.isComplete;
             if (todo.isComplete) {
                 listItem.className += "completed";
+            }else {
+                completedTodoCount++;
```
Comment (seanworkcode) [inline public/main.js]: if ( ! isComplete) -> completedTodoCount++ ??

Comment (cuthullu) [inline public/main.js]: damn, not thinking

**Pull request merged**